src/NHibernate.Test/Logging/LoggerPerformanceTest.cs

```diff
@@ -109,24 +109,26 @@ public void WarnFormat(string format, params object[] args)
 
 		private class MockNHibernateLoggerFactory : INHibernateLoggerFactory
 		{
-			public IInternalLogger2 LoggerFor(string keyName)
+			public INHibernateLogger LoggerFor(string keyName)
```

Comment:
Having seen the factory name, I found it nicer than the 2 suffix and choose a similar naming for the logger.